test/lib/fetch.test.ts

```diff
@@ -23,4 +25,16 @@ describe('fetch', () => {
       superagent,
     }, '/homePageContent', '');
   });
+  it('successfully runs fetchPost', () => {
+    Object.defineProperty(superagent, 'post', {
+      value: jest.fn(),
+    });
+    // const fakeData = [{
+    //   title: 'any',
+    //   comments: 'any',
+    //   type: 'any',
+    // }];
+
+    expect(superagent.post).toBeDefined();
```

Comment:
you would not expect it to be defined because it is defined, you would expect whatever is the outcome of the function call, like a response from superagent.post